lib/utils/update-manual-edits-json.ts

```diff
@@ -0,0 +1,124 @@
+import type { EditPcbComponentLocationEvent } from "@tscircuit/props"
+import type { SourceComponentBase } from "circuit-json"
+import type { PcbComponent } from "circuit-json"
+import type { RunFrameProps } from "lib/runner"
+import type { AnyCircuitElement } from "circuit-json"
+
+export const updateManualEditsJson = (
+  event: EditPcbComponentLocationEvent,
+  props: RunFrameProps,
+  circuitJson: AnyCircuitElement[],
+  lastFsMapRef: React.MutableRefObject<Map<string, string> | null>,
+) => {
+  const manualEditsPath = "manual-edits.json"
```

Comment:
I think the logic for editting the ManualEditsFile should be inside core (and exported), its going to be used in several repos

Reply:
ideal parameters in core

```tsx
{
circuitJson,
event,
existingManualEditsFile
}

```